TheMovieDb/Modules/MovieDetail/MovieDetailLocalDataManager.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  MovieDetailLocalDataManager.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 14/11/21.
+//  
+//
+
+import Foundation
+
+class MovieDetailLocalDataManager: MovieDetailLocalDataManagerInputProtocol {
```

Comment:
why not make this classes that do nothing a Typealias instead ?

Reply:
File deleted, it was a part of the viper template